### PR TITLE
Feat add rocky support

### PR DESCRIPTION
**Pull request by mrichins**

### CODE REVIEW

**Comment by mrichins** [comment]
Whoops, didnt mean to click!

File: cookbooks/fb_yum_repos/libraries/yum_repos_helpers.rb [hunk 1]
```diff
@@ -34,11 +34,15 @@ def self.get_default_gpg_key(_node)
           :default =>
             'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch',
         },
+        :rocky=> {
+          :default =>
+            'https://dl.rockylinux.org/pub/rocky/9/metadata/RPM-GPG-KEY-Rocky-9',
```
Comment (jaymzh) [inline cookbooks/fb_yum_repos/libraries/yum_repos_helpers.rb:39]: There's no local path to the key from an RPM?

**Review by jaymzh — CHANGES_REQUESTED** [review]
Re-opened, but requesting change for question below.

File: cookbooks/fb_yum_repos/libraries/yum_repos_helpers.rb (hunk 1 above)
Comment (mrichins) [reply cookbooks/fb_yum_repos/libraries/yum_repos_helpers.rb:39]: I can double check! This was me experimenting with using fb_cookbooks and trying to get it to play nice with the Rocky family of Linux images. I would want to spend a lot more time working on `fb_helpers` and troubleshooting before formally opening a PR

**Pull request closed without merging**